modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/AlphaDBImport_conf.pm

```diff
@@ -62,16 +62,12 @@ sub default_options {
 
   return {
     %{$self->SUPER::default_options()},
-
-    species => 'homo_sapiens',
     rest_server => 'https://www.ebi.ac.uk/gifts/api/',
-
     user_r => 'ensro',
     password => $ENV{EHIVE_PASS},
     user => 'ensadmin',
-    pipe_db_host => 'mysql-ens-genebuild-prod-7',
-    pipe_db_port => 4533,
-    email_address => $ENV{USER}.'@ebi.ac.uk',
+    pipe_db_host => $self->o('pipe_db_host'),
```

Comment:
To make this parameter mandatory, you can't set the default to itself. either `undef` (making it mandatory init parameter), or just remove it.